### ISSUE TITLE
Test: Test successful app start

### ISSUE THREAD

**Issue by clstaudt** [tuttle-dev/tuttle#218]

**Comment by clstaudt** [comment]
Hi @AmanSal1, could you please open a separate issue to tell us more about the error you are seeing:
- the full terminal output, including your command
- the steps you took to install the app

Thanks!

**Comment by AmanSal1** [comment]
@clstaudt compiled it !! now working fine 

would work on your bug now

**Comment by AmanSal1** [comment]
can you explain in brief about the the issue ?

**Test: Integration test for successful app start #218**

**Comment by clstaudt** [comment]
Good new @AmanSal1 

We welcome your contribution. The task is to implement a test that tests whether the app starts and quits successfully, as part of the test suit in `tuttle_tests`.

Here's how you can proceed:
1. have a look at the unit test suite, run it with `pytest` and check that everything is green
2. add a new test module `tuttle_tests/test_app.py`
3. add a function `test_app_starts_and_quits` and start developing 

Here's how to contribute: 
1. work on a new branch branching off the `dev` branch (e.g. `test-app`)
2. create a new pull request for that branch
3. mark it as draft as long as it is unfinished
4. assign me as a reviewer if ready
5. ask me anything in the discussion of the pull request if you need support

**Comment by AmanSal1** [comment]
@clstaudt as if now there is no module to test that the app starts and quits successfully,so we have to implement a function to check it ? right ?

**Comment by clstaudt** [comment]
@AmanSal1   Correct, currently there is no test in the test suite that would catch every error that prevents the app from starting.

**Comment by AmanSal1** [comment]
Sorry to disturb you again and again but i am bit confused because as i tried to comment out build method in app.py and then run it so it gave me an error .
so basically it is show which module is preventing to start the app

so exatly what do you want ? 

![tuttle](https://user-images.githubusercontent.com/121633121/218160676-8f197fd4-0a8f-433b-84e9-25651d96c59e.PNG)
![tuttle 2](https://user-images.githubusercontent.com/121633121/218160704-f8e93be4-b760-4958-841e-b36d886fc456.PNG)

**Comment by clstaudt** [comment]
@AmanSal1  A unit test that would fail on such an error would be great.

Also, any test that fails when the app fails to start properly.

**Comment by AmanSal1** [comment]
@clstaudt thank you for the explanation i got the clarity will work on it now

**Comment by clstaudt** [comment]
@AmanSal1  Another reason why we need such tests: A developer used a feature from Python 3.10. With Python 3.9 it's a syntax error that prevents the app from starting. No developer noticed because everyone was working with 3.10. That's where automatic tests come in, to quickly test things across environments.

**Comment by AmanSal1** [comment]
@clstaudt yes yes i got your point faced a same issue when i was trying to compile some other github projects .
I am working towards it will get back to you if something positive happens
